web/src/components/FileUpload.jsx

```diff
@@ -48,32 +66,28 @@ export default function FileUpload() {
     setDefaultFileList(fileList);
   };
 
-  const handleCancel = () => {
-    setPreviewVisible(false);
-  };
-
   const handlePreview = async file => {
-    setPreviewVisible(true);
     getBase64(file.originFileObj).then(f => setPreviewImage(f));
+    /**  If you call getBase64(file) you get parameter must be 'blob'. 
+     * If you try to store it with [getBase64(file.originFileObj)] you get the same result on line 62
+     * The commented out lines do not throw exceptions but they do not produce the same key, thus you get an empty result
+     */
+    //TODO: getBase64(file.originFileObj).then(f => setPreviewJSON(annotationResult[md5(f)]));
+    setPreviewJSON(annotationResult[md5(file.name)])
   };
 
   return (
     <div className="container">
       <Upload
         accept="image/*"
         listType="picture-card"
-        customRequest={uploadImage}
+        customRequest={uploadImageToServer}
         onChange={handleOnChange}
         onPreview={handlePreview}
         fileList={defaultFileList}
       >
         {defaultFileList.length >= 8 ? null : <div>Upload Image</div>}
       </Upload>
-      <FileModal
-        onCancel={handleCancel}
-        previewImage={previewImage}
-        previewVisible={previewVisible}
-      />
```

Comment:
So can we delete FileModel.jsx from our components?

Reply:
Yes, currently FileModal.jsx is not in use. I did not delete the file, since there might be a scenario where it can be useful in the future.